.github/actions/python-only/action.yml

```diff
@@ -3,7 +3,7 @@ description: "Prepare the environment to run simple tasks"
 inputs:
   python-version:
     description: "Python version to install"
-    default: "3.9.x"
+    default: "3.9"
```

Comment:
This does the same thing - installs the latest version of python. Reworked according to the prevailing style.